### PR TITLE
Added 54:C4:15 OUI for hikvision camera

### PR DESCRIPTION
**Pull request by jhallard**

### CODE REVIEW

File: cameras/paths.csv [hunk 1]
```diff
@@ -11,7 +11,7 @@ Axis,camera,M3024-LVE,(AC:CC:8E),rtsp://{{username}}:{{password}}@{{ip_address}}
...
 Axis,camera,207W,(00:40:8C),rtsp://{{username}}:{{password}}@{{ip_address}}/mpeg4/media.amp,MPEG4,554,,,,,,,root,,TRUE,http://www.axis.com/files/manuals/um_207w_mw_33158_en_1208.pdf
 Axis,camera,P1214-E,(00:40:8C),rtsp://{{username}}:{{password}}@{{ip_address}}/axis-media/media.amp,H.264,554,,,,,,,root,,,http://www.axis.com/files/manuals/um_axis_P1214e_61497r1_en_1412.pdf
-Hikvision,camera,DS-2CD2032-I;DS-2CD2432F-IW;DS-2CD2035-I;DS-2CD2342WD-I,(C4:2F:90|C0:56:E3|BC:AD:28|44:19:B6|28:57:BE|00:95:69|A4:14:37|20:F4:1B),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/PSIA/streaming/channels/10{{stream}},H.264,554,1;2,,,,2,1,admin,admin,,http://www.hikvision.com/UploadFile/image/2013061411054615328.pdf
+Hikvision,camera,DS-2CD2032-I;DS-2CD2432F-IW;DS-2CD2035-I;DS-2CD2042WD-I,(C4:2F:90|C0:56:E3|BC:AD:28|44:19:B6|28:57:BE|00:95:69|A4:14:37|20:F4:1B|54:C4:15),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/PSIA/streaming/channels/10{{stream}},H.264,554,1;2,,,,2,1,admin,admin,,http://www.hikvision.com/UploadFile/image/2013061411054615328.pdf
```
Comment (jcmaslan) [inline cameras/paths.csv]: Should retain the prior model too (keep DS-2CD2342WD-I)

**Review by jcmaslan — CHANGES_REQUESTED** [review]

**Comment by jcmaslan** [comment]
@jhallard see this request to *add* the model (not replace)?

**Comment by jhallard** [comment]
just saw this, it's back now.

**Comment by jhallard** [comment]
should be good now

**Review by jcmaslan — APPROVED** [review]
LGTM

**Pull request merged**